ansible/host_vars/gateway.yml

```diff
@@ -35,7 +35,7 @@ ntp_allow_hosts: [10.0.0.0/24]
 # firewall role variables
 #########################
 
-in_tcp_port: '{ ssh, https, http, iscsi-target }'
```

Comment:
keep iscsi-target in `in_tcp port`  firewall configuration. Keep support of centralized_san architecture

Reply:
Done